pandas/core/series.py

```diff
@@ -2768,6 +2768,10 @@ def combine(self, other, func, fill_value=None) -> "Series":
         if is_categorical_dtype(self.dtype):
             pass
         elif is_extension_array_dtype(self.dtype):
+            # Everything can be be converted to strings, but we may not want to convert
```

Comment:
realy don't add things here, if you need to modify maybe_cast_to_extesion_array then its ok

this becomes hugely complicated to follow the path otherwise

Reply:
Yes, I would move this into `maybe_cast_to_extension_array` (we will be able to remove this once the "strict" `from_scalars` is implemented, and having the logic in `maybe_cast_to_extension_array` will make it easier to not forget to update that)

Reply:
Ok, found a way to avoid changing series.py at all.